mappings/net/minecraft/client/render/BufferBuilder.mapping

```diff
@@ -8,62 +8,65 @@ CLASS net/minecraft/class_287 net/minecraft/client/render/BufferBuilder
 	FIELD field_1565 format Lnet/minecraft/class_293;
 	FIELD field_1567 drawMode Lnet/minecraft/class_293$class_5596;
 	FIELD field_20884 elementOffset I
-	FIELD field_21594 textured Z
+	FIELD field_21594 canSkipElementChecks Z
+		COMMENT Whether this builder is aware of the vertex format and can skip checks
+		COMMENT for the current target element while building a vertex in {@link
+		COMMENT #vertex(float, float, float, float, float, float, float, float, float, int, int, float, float, float)}.
 	FIELD field_21595 hasOverlay Z
 	FIELD field_27348 sortingPrimitiveCenters [Lnet/minecraft/class_1160;
 	FIELD field_27349 sortingCameraX F
 	FIELD field_27350 sortingCameraY F
 	FIELD field_27351 sortingCameraZ F
-	FIELD field_27352 hasNoVertexBuffer Z
-	FIELD field_32050 MAX_BUFFER_SIZE I
+	FIELD field_27352 indexOnly Z
+	FIELD field_32050 BUFFER_ROUND_SIZE I
 	METHOD <init> (I)V
 		ARG 1 initialCapacity
-	METHOD method_1324 restoreState (Lnet/minecraft/class_287$class_5594;)V
+	METHOD method_1324 restoreSortState (Lnet/minecraft/class_287$class_5594;)V
 		ARG 1 state
 	METHOD method_1326 end ()Lnet/minecraft/class_287$class_7433;
 	METHOD method_1328 begin (Lnet/minecraft/class_293$class_5596;Lnet/minecraft/class_293;)V
 		ARG 1 drawMode
 		ARG 2 format
-	METHOD method_1334 popState ()Lnet/minecraft/class_287$class_5594;
+	METHOD method_1334 getSortState ()Lnet/minecraft/class_287$class_5594;
 	METHOD method_1335 grow (I)V
 		ARG 1 size
 	METHOD method_1343 clear ()V
 	METHOD method_16005 roundBufferSize (I)I
-		ARG 0 amount
+		ARG 0 size
 	METHOD method_22625 ([FII)I
 		ARG 1 a
 		ARG 2 b
 	METHOD method_22892 grow ()V
 	METHOD method_22893 isBuilding ()Z
-	METHOD method_23477 reset ()V
+	METHOD method_23477 discard ()V
 	METHOD method_23918 setFormat (Lnet/minecraft/class_293;)V
 		ARG 1 format
 	METHOD method_31948 sortFrom (FFF)V
 		ARG 1 cameraX
 		ARG 2 cameraY
 		ARG 3 cameraZ
-	METHOD method_31949 createIndexWriter (ILnet/minecraft/class_293$class_5595;)Lit/unimi/dsi/fastutil/ints/IntConsumer;
-	METHOD method_31950 writeSortedIndices (Lnet/minecraft/class_293$class_5595;)V
+	METHOD method_31949 getIndexConsumer (ILnet/minecraft/class_293$class_5595;)Lit/unimi/dsi/fastutil/ints/IntConsumer;
+	METHOD method_31950 putSortedIndices (Lnet/minecraft/class_293$class_5595;)V
 	METHOD method_31954 buildPrimitiveCenters ()[Lnet/minecraft/class_1160;
-	CLASS class_4574 DrawArrayParameters
+	CLASS class_4574 DrawParameters
 		METHOD method_31957 getIndexBufferStart ()I
 		METHOD method_31958 getIndexBufferEnd ()I
 		METHOD method_31961 getIndexBufferLength ()I
 		METHOD method_43429 getVertexBufferPosition ()I
 		METHOD method_43430 getVertexBufferLimit ()I
 		METHOD method_43431 getIndexBufferPosition ()I
 		METHOD method_43432 getIndexBufferLimit ()I
-	CLASS class_5594 State
+	CLASS class_5594 SortState
```

Comment:
```suggestion
	CLASS class_5594 TranslucentPrimives
```
or
```suggestion
	CLASS class_5594 TranslucencySortingData
```
perhaps? sorting will only ever be used for translucency anyways, so specifying it may make it's purpose more obvious.

Reply:
`TranslucencySortingData` looks more accurate than `TranslucentPrimitives`. This class contains more than primitives (such as `vertexCount`) and they are needed to build an index buffer.